src/main/java/frc/robot/Controls.java

```diff
@@ -283,11 +285,24 @@ private void configureSuperStructureBindings() {
                                   superStructure.algaeProcessorScore(
                                       driverController.rightBumper()),
                                   Commands.waitSeconds(0.7),
-                                  getAlgaeIntakeCommand())
+                                  Commands.deferredProxy(
+                                      () ->
+                                          switch (sensors.intakeSensor.inIntakeSwitchable()) {
+                                            case 1 -> Commands.sequence(
```

Comment:
Is there an enum for this?  Try to avoid numerics in a switch to ease future maintenance.